capture-traffic/capture-from-ios.md

```diff
@@ -12,70 +12,102 @@ previous_url: /knowledge-base/configure-ios, /get-started/mobile-traffic/configu
 
-     - In the **SERVER** field, enter the IP address of the Fiddler Everywhere host&mdash;for example, **192.168.0.101** (for demo purposes, we assume this is the IP of the host that runs Fiddler).
+1. Set the Fiddler Everywhere proxy on the iOS device.
 
-     - Enter the Fiddler Everywhere proxy port in the **PORT** field. By default, the port is **8866**.
+    1. On your iOS device, go to **Settings** > **WiFi**.
 
-    >tip With the current setup, you can capture non-secure HTTP traffic. However, if you try to open any HTTPS website, you'll get the "This site's security certificate is not trusted!" error. To fix this issue, trust the Fiddler root certificate.
+    1. Find your current network and click the **i** icon.
 
-1. Open a browser on the iOS device and type the http://ipv4.fiddler:8866 echo service address of Fiddler Everywhere. Follow the Fiddler root certificate link to **download** the Fiddler certificate.
+    1. Scroll to the bottom and choose **Configure Proxy** > **Manual**.
 
-1. On your iOS device, open **General** > **Profile Downloaded** and install the downloaded Fiddler certificate.
+        - In the **SERVER** field, enter the IP address of the Fiddler Everywhere host&mdash;for example, `192.168.0.101` (for demo purposes, we assume this is the IP of the host that runs Fiddler).
+
+        - Enter the Fiddler Everywhere proxy port in the **PORT** field. By default, the port is `8866`.
+
+        >tip With the current setup, you can capture non-secure HTTP traffic. However, if you try to open any HTTPS website, you'll get the "This site's security certificate is not trusted!" error. To fix this issue, trust the Fiddler root certificate.
 
-1. (iOS 10.3+) Go to **Settings** > **General** > **About** > **Certificate Trust Settings** and enable full trust for the **Fiddler_Root_Certificate_Authority** certificate.
 
 >important The last step is crucial, and HTTPS browsing will fail if the certificate is not trusted! Newer iOS versions will show that the certificate is successfully installed. Still, HTTPS sites will fail to open due to security errors if the certificate is not explicitly enabled/fully trusted.
 
 
-## Configuring iOS Simulator
+## Configuring the Fiddler proxy on iOS Emulator
+
+The later versions of the iOS simulators, accessible through XCode, are using the macOS system proxy by default. 
+
+1. In Fiddler Everywhere, open the **Traffic** pane and enable the system capturing mode by toggling ON the **System Proxy** switch.
+
+    >important The iOS simulators are automatically detecting the macOS proxy settings. However, not all simulators are not dynamically detecting changes in the OS proxy settings. This step must execute before the iOS simulator starts!
+
+1. In Fiddler Everywhere, go to **Settings** > **HTTPS** and enable **Capture HTTPS traffic**.
+
+1. In Fiddler Everywhere, go to **Settings** > **Connections** and enable **Allow remote devices to connect**.
```

Comment:
`Allow remote computers to connect`

Reply:
already fixed to **Allow remote devices to connect**